packages/css/stories/selectable/selectable.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { storiesOfSelectable, SelectableArgs, SelectableTemplateFn } from '@dso-toolkit/stories';
+import { storiesOf } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+// @ts-ignore
+import readme from './readme.md';
+
+const template: SelectableTemplateFn<TemplateResult> = ({
+  type,
+  id,
+  name,
+  label,
+  value,
+  required,
+  invalid,
+  checked,
+  disabled,
+  onChange
+}: SelectableArgs) => html`
+  <div class="dso-selectable">
+    <input
+      type=${type}
+      id=${id}
+      value=${value}
+      name=${ifDefined(name)}
+      aria-invalid=${ifDefined(invalid)}
```

Comment:
Hier lijkt het of hij alleen de `aria-invalid` zet als attribuut, maar ik dacht dat we die als `aria-invalid="true"` of `aria-invalid="false"` moeten neerzetten.

Reply:
Dat doet ie ook. Je kan 't lokaal draaien door de branch uit te checken, `yarn` en dan `yarn start:all` te draaien.